server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/FragmentInstance.java

```diff
@@ -129,30 +140,24 @@ public static FragmentInstance deserializeFrom(ByteBuffer buffer) {
     FragmentInstance fragmentInstance =
         new FragmentInstance(
             planFragment, Integer.parseInt(id.getInstanceId()), timeFilter, queryType);
-    RegionReplicaSet regionReplicaSet = new RegionReplicaSet();
-    try {
-      regionReplicaSet.deserializeImpl(buffer);
-    } catch (IOException e) {
-      e.printStackTrace();
-    }
-    Endpoint endpoint = new Endpoint();
-    endpoint.deserializeImpl(buffer);
-    fragmentInstance.dataRegion = regionReplicaSet;
-    fragmentInstance.hostEndpoint = endpoint;
+    fragmentInstance.regionReplicaSet = RegionReplicaSet.deserializeImpl(buffer);
+    fragmentInstance.hostEndpoint = Endpoint.deserializeImpl(buffer);
 
     return fragmentInstance;
   }
 
   @Override
   public void serializeRequest(ByteBuffer buffer) {
+    buffer.mark();
```

Comment:
why use this? Is there anywhere use `buffer.reset()`?

Reply:
can remove. I will fix it next pr